assets/out

```diff
@@ -33,6 +33,8 @@ password=$(jq -r '.source.password // ""' < $payload)
 repository=$(jq -r '.source.repository // ""' < $payload)
 ca_certs=$(jq -r '.source.ca_certs // []' < $payload)
 client_certs=$(jq -r '.source.client_certs // []' < $payload)
+max_concurrent_downloads=$(jq -r '.source.max_concurrent_downloads // 3' < $payload || echo "3")
```

Comment:
Can you do the same as above?

Reply:
Is three the default the docker always provides? This is backwards change that would make other's pipelines go slower until they updated to max a higher max. Maybe we should make this optional, as in, you provide a value use it, no value, don't set any min/max.

Reply:
Yes, docker default for download is 3, upload is - for no obvious reason - defaulting to 5

Reply:
Is the `|| echo "3"` necessary? The `// 3` should default the value to `3` if it's not present.

Reply:
No it is not but it has been removed in https://github.com/concourse/docker-image-resource/pull/144/commits/99994f0fc18f7d7e72f713013df4034e41e30c1b